src/components/visx/TimeSeriesChart.tsx

```diff
@@ -158,14 +156,14 @@ export const TimeSeriesChart = <Datum extends {}>({
   useAnimationFrame(
     (elapsedMilliseconds) => {
       if (zoomDomainAnimateTo) {
-        setZoomDomain((zoomDomain) => {
-          if (!zoomDomain) return zoomDomain;
+        setZoomDomain((oldZd) => {
```

Comment:
[nit/non-blocking] can we call this `oldZoomDomain`? there's plenty of context clues but it's not immediately obvious what `oldZd` is

Reply:
good call